opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -189,17 +215,36 @@ impl<R: RuntimeChannel> LogProcessor for BatchLogProcessor<R> {
     }
 
     fn force_flush(&self) -> LogResult<()> {
-        let (res_sender, res_receiver) = oneshot::channel();
-        self.message_sender
-            .try_send(BatchMessage::Flush(Some(res_sender)))
+        if self.is_shutdown.load(std::sync::atomic::Ordering::Relaxed) {
+            otel_warn!(
```

Comment:
nit - no need to emit warning here, as we are already returning result to the application. If required for debugging purpose, we can make it otel_debug.

Reply:
Removed warning.